testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js

```diff
@@ -431,6 +431,37 @@ QUnit.module('general', {}, () => {
             browser.version = origBrowser.version;
         }
     });
+
+    QUnit.test('editors has no collapsed class in IE11 if the editor has custom display style', function(assert) {
+        const origBrowser = $.extend({}, browser);
+        browser.msie = true;
+        browser.version = '11.0';
+        const $element = $('#texteditor');
+        try {
+            $element.css('display', 'inline-block');
+            const $textEditor = $element.dxTextEditor({});
+            assert.notOk($textEditor.hasClass('dx-texteditor-collapsed'));
+        } finally {
+            browser.msie = origBrowser.msie;
+            browser.version = origBrowser.version;
+            $element.css('display', 'block');
```

Comment:
Looks like this is not needed since QUnit fixture automatically restores markup after test completion.